webpack.dev.js

```diff
@@ -0,0 +1,16 @@
+const merge = require('webpack-merge');
+const common = require('./webpack.common.js');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+module.exports = merge(common, {
```

Comment:
fica o questionamento, é necessário separar dev e prod levando em conta que usamos o storybook?

Reply:
Sim, pois se for necessário buildar esse projeto exportá-lo já possui todas as configurações necessárias, tanto pra produção (para um npm da vida) como para desenvolvimento